src/Goerli-DssSpellCollateral.sol

```diff
@@ -32,135 +32,32 @@ contract DssSpellCollateralAction {
     //
 
     // --- Math ---
-    // uint256 constant THOUSAND   = 10 ** 3;
-    // uint256 constant MILLION    = 10 ** 6;
-    // uint256 constant BILLION    = 10 ** 9;
+    uint256 constant MILLION = 10 ** 6;
```

Comment:
Do we want this in this file or Goerli-DssSpell.sol?

Reply:
It has to be in this file since they use it here.